txcache/trackedBlock.go

```diff
@@ -31,6 +31,41 @@ func newTrackedBlock(
 	}
 }
 
+func (tb *trackedBlock) createOrUpdateVirtualRecords(
+	session SelectionSession,
+	skippedSenders map[string]struct{},
+	sendersInContinuityWithSessionNonce map[string]struct{},
+	accountPreviousBreadcrumb map[string]*accountBreadcrumb,
+	virtualAccountsByAddress map[string]*virtualAccountRecord,
+) error {
+	for address, breadcrumb := range tb.breadcrumbsByAddress {
+		_, ok := skippedSenders[address]
+		if ok {
+			continue
+		}
+
+		accountState, err := session.GetAccountState([]byte(address))
+		if err != nil {
+			log.Debug("selectionTracker.createVirtualSelectionSession",
```

Comment:
Let's make a note about this (a check we have to make during the integration phase): make sure we properly handle accounts which don't yet exist in the non-virtual-session (backed by the accounts database on-chain), but get created within previously proposed (not yet executed!) blocks.

E.g. Alice is unknown to the blockchain (to the non-virtual session), but it's the receiver of a very recent transaction - maybe `GetAccountState` already works fine in that case?